pkg/store/proxy.go

```diff
@@ -591,12 +604,17 @@ func storeMatchDebugMetadata(s Client, storeDebugMatchers [][]*labels.Matcher) (
 		return true, ""
 	}
 
+	clientType, addr := s.StoreInfo()
+	if clientType != Remote {
+		return false, "the store is not remote, cannot match __address__"
```

Comment:
Hmm, you can - just it can be `local` or some derivative like `local-tenant1`, no? (:

Reply:
Similar to my comment above, I understand `__address__` to be applicable really only to remote stores.